tests/compute/frem.slang

```diff
@@ -2,12 +2,12 @@
 
 //TEST(compute):COMPARE_COMPUTE: -shaderobj
 //TEST(compute):COMPARE_COMPUTE:-cpu -shaderobj
-//TEST(compute):COMPARE_COMPUTE:-vk -shaderobj
+//TEST(compute):COMPARE_COMPUTE:-vk -shaderobj -emit-spirv-directly
```

Comment:
You can add `-output-using-type` to this `//TEST` lines so that floating point values will be written as floating points instead of binaries. You can search for it in the repo to see how it is used in other tests.